From 5b81100f0e3680bdbea9caac065630335af5bbff Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 4 Jul 2022 18:10:23 +0100
Subject: [PATCH] media: i2c: ov9281: Correct min/def vts for 640x400

A VTS of 421 in the 640x400 mode only streams at 130.2fps
instead of the expected ~261fps. This appears to be an invalid
value for the sensor, as a VTS of 422 works fine to give 259.79fps.

Set the minimum (and default) to 422.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/media/i2c/ov9281.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/ov9281.c b/drivers/media/i2c/ov9281.c
index 0a5f9e0927ae..2fb40e63c585 100644
--- a/drivers/media/i2c/ov9281.c
+++ b/drivers/media/i2c/ov9281.c
@@ -369,7 +369,7 @@ static const struct ov9281_mode supported_modes[] = {
 		.height = 400,
 		.exp_def = 0x0320,
 		.hts_def = 0x05b0,
-		.vts_def = 421,
+		.vts_def = 422,
 		.crop = {
 			.left = 0,
 			.top = 0,
-- 
2.32.0

